frontend/src/app/util/keyboard.cljs

```diff
@@ -4,37 +4,45 @@
 ;;
 ;; Copyright (c) UXBOX Labs SL
 
-(ns app.util.keyboard)
+(ns app.util.keyboard
+  (:require
+   [app.config :as cfg]))
 
 (defn is-key?
-  [key]
-  (fn [e]
+  [^string key]
+  (fn [^js e]
     (= (.-key e) key)))
 
 (defn ^boolean alt?
-  [event]
+  [^js event]
   (.-altKey event))
 
 (defn ^boolean ctrl?
-  [event]
+  [^js event]
   (.-ctrlKey event))
 
 (defn ^boolean meta?
-  [event]
+  [^js event]
   (.-metaKey event))
 
 (defn ^boolean shift?
-  [event]
+  [^js event]
   (.-shiftKey event))
 
+(defn ^boolean mod?
+  [^js event]
+  (if (cfg/check-platform? :macos)
+    (meta? event)
+    (ctrl? event)))
+
 (def esc? (is-key? "Escape"))
 (def enter? (is-key? "Enter"))
 (def space? (is-key? " "))
 (def up-arrow? (is-key? "ArrowUp"))
 (def down-arrow? (is-key? "ArrowDown"))
 (def altKey? (is-key? "Alt"))
-(def ctrlKey? (or (is-key? "Control")
-                  (is-key? "Meta")))
+(def ctrlKey? (is-key? "Control"))
+(def metaKey? (is-key? "Meta"))
```

Comment:
At some point we should rename that camelCase names to lisp/kebab case

Reply:
I'll change it